src/main/java/org/apache/xml/security/resource/xmlsecurity_en.properties

```diff
@@ -111,7 +111,9 @@ KeyStore.registerStore.register = Registration error for type {0}
 KeyValue.IllegalArgument = Cannot create a {0} from {1}
 KeyDerivation.TooShortParameter = Key derivation parameter {0} is too short
 KeyDerivation.InvalidParameter = Key derivation parameter {0} is illegal
+KeyDerivation.InvalidParametersType = Algorithm {0} requires key derivation parameters of type {1} to be provided.

```

Comment:
Nit: most of the other error messages end in a period, so suggest removing it.

Reply:
Done